### ISSUE TITLE
Build uses CRAN snapshot instead of current packages

### ISSUE THREAD

**Issue by Wytamma** [Wytamma/rstudio-hpc#4]
When the Rstudio container is built it sets the repository to a snapshot so that package versions don't get mess up (i.e. it will always have the versions that you installed a build time). However, if the user installs new packages the repository snapshot is stilled used so you can't update packages past the snapshot date. 

To solve you can use the `repos` arg when installing packages and specify the live CRAN. 

`install.packages("ampir", repos='http://cran.r-project.org')`

You can also set a system wide default repo with the following code.

```R
local({r <- getOption("repos")
       r["CRAN"] <- "http://cran.r-project.org" 
       options(repos=r)
})
```

I think I should add this solution to the build file although I'm not sure what the idiomatic solution should be?

**Comment by Wytamma** [comment]
I found a related issue on rocker that might have the solution (https://github.com/rocker-org/rocker-versioned/issues/102). 

`RUN echo "options(repos = c(CRAN = 'https://cran.rstudio.org'))" >> /usr/local/lib/R/etc/Rprofile.site`

**Comment by Wytamma** [comment]
Solved in #5
